ai_flow/endpoint/server/server_config.py

```diff
@@ -130,4 +131,20 @@ def set_wait_for_server_started_timeout(self, wait_for_server_started_timeout):
         self['wait_for_server_started_timeout'] = wait_for_server_started_timeout
 
     def get_wait_for_server_started_timeout(self):
-        return self.get('wait_for_server_started_timeout')
+        if 'wait_for_server_started_timeout' in self:
+            return self.get('wait_for_server_started_timeout')
+        else:
+            return 5.0
+
+
+def get_configuration():
```

Comment:
We can use get_configuration method in `settings.py`

Reply:
Removed